### ISSUE TITLE
Test case failing: WebSocketTests.WebSocket_Https_Duplex_TextStreamed

### ISSUE THREAD

**Issue by StephenBonikowsky** [dotnet/wcf#2348]
This test case is failing on all non-Windows OSes in the auto update PR for Master.

Error...
```
Stacktrace
MESSAGE:
System.InvalidOperationException : An error occurred when creating the WebSocket with the factory of type 'CoreClrClientWebSocketFactory'. See the inner exception for details.
\n---- System.Net.WebSockets.WebSocketException : Unable to connect to the remote server
\n-------- System.Net.Http.HttpRequestException : An error occurred while sending the request.
\n------------ System.IO.IOException : The encryption operation failed, see inner exception.
\n---------------- System.MissingMethodException : Method not found: 'Void* System.Buffers.MemoryHandle.get_PinnedPointer()'.
+++++++++++++++++++
STACK TRACE:
at System.ServiceModel.Channels.ClientWebSocketTransportDuplexSessionChannel.<CreateWebSocketWithFactoryAsync>d__17.MoveNext() in /mnt/j/workspace/dotnet_wcf/master/outerloop_ubuntu14.04_release_prtest/src/System.Private.ServiceModel/src/System/ServiceModel/Channels/ClientWebSocketTransportDuplexSessionChannel.cs:line 225 
--- End of stack trace from previous location where exception was thrown --- 
at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() 
at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) 
at System.ServiceModel.Channels.ClientWebSocketTransportDuplexSessionChannel.<OnOpenAsync>d__12.MoveNext() in /mnt/j/workspace/dotnet_wcf/master/outerloop_ubuntu14.04_release_prtest/src/System.Private.ServiceModel/src/System/ServiceModel/Channels/ClientWebSocketTransportDuplexSessionChannel.cs:line 94 
--- End of stack trace from previous location where exception was thrown --- 
at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() 
at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) 
at System.ServiceModel.Channels.CommunicationObject.<OnOpenAsyncInternal>d__80.MoveNext() in /mnt/j/workspace/dotnet_wcf/master/outerloop_ubuntu14.04_release_prtest/src/System.Private.ServiceModel/src/System/ServiceModel/Channels/CommunicationObject.cs:line 461 
--- End of stack trace from previous location where exception was thrown --- 
at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() 
at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) 
at System.ServiceModel.Channels.CommunicationObject.<System-ServiceModel-IAsyncCommunicationObject-OpenAsync>d__79.MoveNext() in /mnt/j/workspace/dotnet_wcf/master/outerloop_ubuntu14.04_release_prtest/src/System.Private.ServiceModel/src/System/ServiceModel/Channels/CommunicationObject.cs:line 436 
--- End of stack trace from previous location where exception was thrown --- 
at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() 
at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) 
at System.ServiceModel.Channels.CommunicationObject.<OpenAsyncInternal>d__78.MoveNext() in /mnt/j/workspace/dotnet_wcf/master/outerloop_ubuntu14.04_release_prtest/src/System.Private.ServiceModel/src/System/ServiceModel/Channels/CommunicationObject.cs:line 410 
--- End of stack trace from previous location where exception was thrown --- 
at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() 
at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) 
at System.ServiceModel.Channels.CommunicationObject.Open(TimeSpan timeout) in /mnt/j/workspace/dotnet_wcf/master/outerloop_ubuntu14.04_release_prtest/src/System.Private.ServiceModel/src/System/ServiceModel/Channels/CommunicationObject.cs:line 405 
at System.Runtime.TaskHelpers.<CallActionAsync>d__17`1.MoveNext() in /mnt/j/workspace/dotnet_wcf/master/outerloop_ubuntu14.04_release_prtest/src/System.Private.ServiceModel/src/Internals/System/Runtime/TaskHelpers.cs:line 312 --- End of stack trace from previous location where exception was thrown --- 
at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() 
at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) 
at System.ServiceModel.Channels.CommunicationObject.<OpenOtherAsync>d__109.MoveNext() in /mnt/j/workspace/dotnet_wcf/master/outerloop_ubuntu14.04_release_prtest/src/System.Private.ServiceModel/src/System/ServiceModel/Channels/CommunicationObject.cs:line 897 
--- End of stack trace from previous location where exception was thrown --- 
at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() 
at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) 
at System.ServiceModel.Channels.ServiceChannel.<OnOpenAsync>d__151.MoveNext() in /mnt/j/workspace/dotnet_wcf/master/outerloop_ubuntu14.04_release_prtest/src/System.Private.ServiceModel/src/System/ServiceModel/Channels/ServiceChannel.cs:line 1364 
--- End of stack trace from previous location where exception was thrown --- 
at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() 
at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) 
at System.ServiceModel.Channels.CommunicationObject.<OnOpenAsyncInternal>d__80.MoveNext() in /mnt/j/workspace/dotnet_wcf/master/outerloop_ubuntu14.04_release_prtest/src/System.Private.ServiceModel/src/System/ServiceModel/Channels/CommunicationObject.cs:line 461 
--- End of stack trace from previous location where exception was thrown --- 
at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() 
at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) 
at System.ServiceModel.Channels.CommunicationObject.<System-ServiceModel-IAsyncCommunicationObject-OpenAsync>d__79.MoveNext() in /mnt/j/workspace/dotnet_wcf/master/outerloop_ubuntu14.04_release_prtest/src/System.Private.ServiceModel/src/System/ServiceModel/Channels/CommunicationObject.cs:line 436 
--- End of stack trace from previous location where exception was thrown --- 
at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() 
at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) 
at System.ServiceModel.Channels.CommunicationObject.<OpenAsyncInternal>d__78.MoveNext() in /mnt/j/workspace/dotnet_wcf/master/outerloop_ubuntu14.04_release_prtest/src/System.Private.ServiceModel/src/System/ServiceModel/Channels/CommunicationObject.cs:line 410 
--- End of stack trace from previous location where exception was thrown --- 
at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() 
at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) 
at System.ServiceModel.Channels.ServiceChannel.CallOpenOnce.System.ServiceModel.Channels.ServiceChannel.ICallOnce.Call(ServiceChannel channel, TimeSpan timeout) in /mnt/j/workspace/dotnet_wcf/master/outerloop_ubuntu14.04_release_prtest/src/System.Private.ServiceModel/src/System/ServiceModel/Channels/ServiceChannel.cs:line 1982 
at System.ServiceModel.Channels.ServiceChannel.CallOnceManager.CallOnce(TimeSpan timeout, CallOnceManager cascade) in /mnt/j/workspace/dotnet_wcf/master/outerloop_ubuntu14.04_release_prtest/src/System.Private.ServiceModel/src/System/ServiceModel/Channels/ServiceChannel.cs:line 2073 
at System.ServiceModel.Channels.ServiceChannel.EnsureOpened(TimeSpan timeout) in /mnt/j/workspace/dotnet_wcf/master/outerloop_ubuntu14.04_release_prtest/src/System.Private.ServiceModel/src/System/ServiceModel/Channels/ServiceChannel.cs:line 507 
at System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout) in /mnt/j/workspace/dotnet_wcf/master/outerloop_ubuntu14.04_release_prtest/src/System.Private.ServiceModel/src/System/ServiceModel/Channels/ServiceChannel.cs:line 764 
at System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(MethodCall methodCall, ProxyOperationRuntime operation) in /mnt/j/workspace/dotnet_wcf/master/outerloop_ubuntu14.04_release_prtest/src/System.Private.ServiceModel/src/System/ServiceModel/Channels/ServiceChannelProxy.cs:line 369 
at System.ServiceModel.Channels.ServiceChannelProxy.Invoke(MethodInfo targetMethod, Object[] args) in /mnt/j/workspace/dotnet_wcf/master/outerloop_ubuntu14.04_release_prtest/src/System.Private.ServiceModel/src/System/ServiceModel/Channels/ServiceChannelProxy.cs:line 138 
--- End of stack trace from previous location where exception was thrown --- 
at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() 
at System.Reflection.DispatchProxyGenerator.Invoke(Object[] args) 
at generatedProxy_1.DownloadStream() 
at WebSocketTests.WebSocket_Https_Duplex_TextStreamed() in /mnt/j/workspace/dotnet_wcf/master/outerloop_ubuntu14.04_release_prtest/src/System.Private.ServiceModel/tests/Scenarios/Extensibility/WebSockets/WebSocketTests.4.1.0.cs:line 303 
----- Inner Stack Trace ----- 
at System.Net.WebSockets.WebSocketHandle.<ConnectAsyncCore>d__22.MoveNext() in /root/corefx-1080133/src/System.Net.WebSockets.Client/src/System/Net/WebSockets/WebSocketHandle.Managed.cs:line 193 
--- End of stack trace from previous location where exception was thrown --- 
at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() 
at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) 
at System.Net.WebSockets.ClientWebSocket.<ConnectAsyncCore>d__16.MoveNext() in /root/corefx-1080133/src/System.Net.WebSockets.Client/src/System/Net/WebSockets/ClientWebSocket.cs:line 157 
--- End of stack trace from previous location where exception was thrown --- 
at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() 
at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) 
at System.ServiceModel.Channels.CoreClrClientWebSocketFactory.<CreateWebSocketAsync>d__0.MoveNext() in /mnt/j/workspace/dotnet_wcf/master/outerloop_ubuntu14.04_release_prtest/src/System.Private.ServiceModel/src/System/ServiceModel/Channels/CoreClrClientWebSocketFactory.cs:line 36 
--- End of stack trace from previous location where exception was thrown --- 
at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() 
at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) 
at System.ServiceModel.Channels.ClientWebSocketTransportDuplexSessionChannel.<CreateWebSocketWithFactoryAsync>d__17.MoveNext() in /mnt/j/workspace/dotnet_wcf/master/outerloop_ubuntu14.04_release_prtest/src/System.Private.ServiceModel/src/System/ServiceModel/Channels/ClientWebSocketTransportDuplexSessionChannel.cs:line 216 
----- Inner Stack Trace ----- 
at System.Net.Http.HttpConnection.<SendAsync>d__37.MoveNext() in /root/corefx-1080133/src/System.Net.Http/src/System/Net/Http/Managed/HttpConnection.cs:line 413 
--- End of stack trace from previous location where exception was thrown --- 
at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() 
at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) 
at System.Net.Http.HttpConnectionHandler.<SendAsyncWithAwaitedConnection>d__4.MoveNext() in /root/corefx-1080133/src/System.Net.Http/src/System/Net/Http/Managed/HttpConnectionHandler.cs:line 43 
--- End of stack trace from previous location where exception was thrown --- 
at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() 
at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) 
at System.Net.Http.CookieHandler.<SendAsync>d__3.MoveNext() in /root/corefx-1080133/src/System.Net.Http/src/System/Net/Http/Managed/CookieHandler.cs:line 31 
--- End of stack trace from previous location where exception was thrown --- 
at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() 
at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) 
at System.Net.Http.AutoRedirectHandler.<SendAsync>d__5.MoveNext() in /root/corefx-1080133/src/System.Net.Http/src/System/Net/Http/Managed/AutoRedirectHandler.cs:line 65 
--- End of stack trace from previous location where exception was thrown --- 
at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() 
at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) 
at System.Net.WebSockets.WebSocketHandle.<ConnectAsyncCore>d__22.MoveNext() in /root/corefx-1080133/src/System.Net.WebSockets.Client/src/System/Net/WebSockets/WebSocketHandle.Managed.cs:line 138 
----- Inner Stack Trace ----- 
at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() 
at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) 
at System.Net.Security.SslStreamInternal.<<WriteAsyncInternal>g__ExitWriteAsync|32_0>d`1.MoveNext() in /root/corefx-1080133/src/System.Net.Security/src/System/Net/Security/SslStreamInternal.cs:line 285 
--- End of stack trace from previous location where exception was thrown --- 
at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() 
at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) 
at System.Net.Http.HttpConnection.<SendAsync>d__37.MoveNext() in /root/corefx-1080133/src/System.Net.Http/src/System/Net/Http/Managed/HttpConnection.cs:line 260 
----- Inner Stack Trace ----- 
at Interop.OpenSsl.Encrypt(SafeSslHandle context, ReadOnlyMemory`1 input, Byte[]& output, SslErrorCode& errorCode) 
at System.Net.Security.SslStreamPal.EncryptDecryptHelper(SafeDeleteContext securityContext, ReadOnlyMemory`1 input, Int32 offset, Int32 size, Boolean encrypt, Byte[]& output, Int32& resultSize) in /root/corefx-1080133/src/System.Net.Security/src/System/Net/Security/SslStreamPal.Unix.cs:line 193
```

**Comment by StephenBonikowsky** [comment]
MemoryHandle’s PinnedPointer property was renamed to Pointer, but that hasn’t propagated through corefx yet.
Waiting for https://github.com/dotnet/corefx/pull/24777 to be merged.

**Comment by zhenlan** [comment]
@StephenBonikowsky can this be closed now since dotnet/corefx#24777 is merged?
